decrypt_oracle/src/aws_encryption_sdk_decrypt_oracle/app.py

```diff
@@ -0,0 +1,68 @@
+"""Decrypt Oracle using the AWS Encryption SDK for Python."""
+import json
+import logging
+import os
+
+import aws_encryption_sdk
+from aws_encryption_sdk.key_providers.kms import KMSMasterKeyProvider
+from chalice import Chalice, Response
+
+from .key_providers.counting import CountingMasterKey
+from .key_providers.null import NullMasterKey
+
+CHALICE_DEBUG = os.environ.get("CHALICE_DEBUG", "no") == "yes"
+APP = Chalice(app_name="aws-encryption-sdk-decrypt-oracle", debug=CHALICE_DEBUG)
+APP.log.setLevel(logging.DEBUG)
+
+try:  # Python 3.5.0 and 3.5.1 have incompatible typing modules
```

Comment:
this should only be building for 3.6.  Why do we need this?

Reply:
That...is a fair point. I've gotten so much into the habit of having to support everything that I completely forgot that by the nature of the target environment this is strictly 3.6.